### PR TITLE
Fix callback when result is false

### PR DESCRIPTION
**Pull request by DavOnGit**
faceyspacey/redux-first-router/issues/321

@ScriptedAlchemy @hedgepigdaniel

### CODE REVIEW

File: modules/createTransitionManager.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const createTransitionManager = () => {
         }
       } else {
         // Return false from a transition hook to cancel the transition.
-        callback(result !== false)
+        callback(false)
```
Comment (hedgepigdaniel) [inline modules/createTransitionManager.js]: Won't this result in always blocking the transition, even if `confirmLeave` from RFR (which I think is `prompt`) returns `true`? I would have thought

```javascript
callback(Boolean(result));
```
would be what is needed.

**Review by hedgepigdaniel — COMMENTED** [review]
Did you try installing this patched version as the dependency of RFR and running all the tests?

I think that may uncover something.

File: modules/createTransitionManager.js (hunk 1 above)
Comment (ScriptedAlchemy) [reply modules/createTransitionManager.js]: I can implement that if @DavOnGit @hedgepigdaniel  agree?

Comment (ScriptedAlchemy) [reply modules/createTransitionManager.js]: Update the PR and if possible, make a test for this case / extend an existing one

Comment (DavOnGit) [reply modules/createTransitionManager.js]: I'll try to test it next days, ty!

Comment (ScriptedAlchemy) [reply modules/createTransitionManager.js]: Thanks

**Pull request merged**